GRIB2_CodeFlag_4_103_CodeTable_en.csv

```diff
@@ -0,0 +1,9 @@
+Title_en,SubTitle_en,CodeFlag,Value,MeaningParameterDescription_en,Note_en,noteIDs,UnitComments_en,Status
+Spatial vicinity type,,0,,Circle[m],(see Note),225a,,Operational
+Spatial vicinity type,,1,,"Rectangle [m,m]",(see Note),225a,,Operational
+Spatial vicinity type,,2,,Square [m],(see Note),225a,,Operational
+Spatial vicinity type,,3,,"Wedge [m, degree, degree]",(see Note),225a,,Operational
+Spatial vicinity type,,4,,"Span of grid boxes centered around grid box i,j [x,y]",(see Note),225a,,Operational
```

Comment:
This differs from the proposal, "Number of grid cells [-,-]

Reply:
I have reformulated the entry so that it is clearer